src/analysis/complex/real_deriv.lean

```diff
@@ -1,18 +1,36 @@
 /-
 Copyright (c) Sébastien Gouëzel. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Sébastien Gouëzel
+Authors: Sébastien Gouëzel, Yourong Zang
 -/
 import analysis.calculus.times_cont_diff
-import analysis.complex.basic
+import analysis.complex.conformal
+import analysis.calculus.conformal
 
 /-! # Real differentiability of complex-differentiable functions
 
 `has_deriv_at.real_of_complex` expresses that, if a function on `ℂ` is differentiable (over `ℂ`),
 then its restriction to `ℝ` is differentiable over `ℝ`, with derivative the real part of the
 complex derivative.
--/
 
+`differentiable_at.conformal_at` states that a real-differentiable function with a nonvanishing
+differential from the complex plane into an arbitrary complex-normed space is conformal at a point
+if it's holomorphic at that point. This is a version of Cauchy-Riemann equations.
+
+`conformal_at_iff_differentiable_at_or_differentiable_at_comp_conj` proves that a real-differential
+function with a nonvanishing differential between the complex plane is conformal at a point if and
+only if it's holomorphic or antiholomorphic at that point.
+
+## TODO
+
+* The classical form of Cauchy-Riemann equations
+* A function is holomorphic or antiholomorphic on a domain `u` if it's conformal at every point in
+that domain.
```

Comment:
```suggestion
* On a connected open set `u`, a function which is `conformal_at` each point is either holomorphic
throughout or antiholomorphic throughout.
```
Your description was correct, but do you mind this rephrasing, which emphasizes that the point is the connectedness?

Reply:
This one is much clearer. Thank you!